docs/how-to/build-policies/rebac/overview.mdx

```diff
@@ -3,124 +3,106 @@ sidebar_position: 1
-("Friends of Friends").
+:::tipExample
+The simples example for ReBAC is a file system (e.g. [Google Drive](https://docs.permit.io/modeling/google-drive)).
+![Google Drive](/ui-videos/rebac/1.png)
+This graph displays a folder titled `Bob’s Files`. Within that folder, there are two additional folders - `Bob’s Docs` and `Bob’s Pics`. Inside each of these folders are several files.
+:::
 
-### Example #2: Collaborative Document Editing Platform
+To understand how ReBAC helps us manage access to these files, we need to understand two basic terms:
 
-Alice and Bob are working on a project report in Google Docs. Alice creates the main document, and Bob adds supplementary
-material as an appendix in a separate document. Alice then links Bob's appendix to the main report. Now, anyone with
-permission to edit Alice's main report should also automatically gain permission to edit Bob's appendix because of the
-linkage (or relationship) between the two documents.
-
-Check out this [Google Drive example](/modeling/google-drive) being modeled with Permit.
-
-## Basic definitions
-
-### Role Assignment
-
-Role Assignment refers to the act of designating specific roles to individual users or entities, effectively granting them
-the permissions associated with those roles. Roles represent a collection of permissions. By assigning a user a particular role,
-you're essentially giving them access to all the permissions bundled within that role.
-This mechanism simplifies access management, especially in large systems or organizations, as administrators only need to
-manage roles rather than each specific permission for every user.
-
-#### Example
-
-In a company, new employees in the marketing department might be assigned the "Marketing" role, which provides access to
-relevant databases, software, and internal websites without having to configure each access permission individually.
-
-### Resource Role
-
-A Resource Role is a specialized role that's specifically tied to a particular resource.
-This means that the role, and the permissions it carries, is relevant only in the context of that specific resource.
-By creating roles based on resources, organizations can have fine-grained control over who can access and modify specific items.
+### Role Derivation
 
-#### Example
+ReBAC allows us to derive authorization policies based on existing application-level relationships. Put in the simplest way, it allows us to create a policy like this:
 
-In a cloud storage solution like Google Drive, a "Document Editor" role for a specific document might grant permissions to edit
-and comment on that document. However, having the "Document Editor" role for one document doesn't mean the user can edit all
-documents—only the specific one they're assigned to.
+```
+A `user` who is assigned the role of an `Owner` on a folder will also get the `Owner` role on every `file` within that `folder`.
```

Comment:
This text is half-hidden as it is a long line. Worth putting it in a quote or something.